### ISSUE TITLE
save path

### ISSUE THREAD

**Issue by tonycaso** [tennisonchan/fabric-brush#1]
Awesome plugin, thanks for developing it!

It would be great if it could be integrated completly with fabric, as after leaving drawing mode the painting dissapears. It has to be with creating svg path, hasn't it?

Very beatiful indeed.

**Comment by adsgm** [comment]
Same here, the InkBrush is just amazing, but when I try to save my canvas to PNG (with canvas.toDataURL) the drawing disappears.

Great work!

**Comment by elendee** [comment]
agreed awesome plugin, learned a lot from it.  any progress on this point though?  I'm going to look into it over the next couple days.  A little tricky as I imagine every "stroke of the brush" needs to be added as a locked fabric group.  But it should be possible...

**Comment by elendee** [comment]
well.. I did manage to get it working in a minimal way, but to draw all those shapes as fabric objects crashed my browser immediately.  So, you can make your own custom brushes as fabric objects I think, but not THESE brushes exactly.  If you want to experiment, look in "fabric-brush.js", copy out the brush you're interested in to a stand-alone script, and replace all instances of "ctx" with your fabric canvas, basically.
